### PR TITLE
refactor(authui): upgrade to Firebase Auth v20.0.0

### PR DESCRIPTION
**Pull request by rosariopfernandes**
This PR should update FirebaseUI auth to use the new `PhoneAuthProvider.verifyPhoneNumber(PhoneAuthOptions)` method.
~- update the Firebase BoM to v26.0.0, so we can use Firebase Auth v20.0.0~

Fixes firebase/quickstart-android#1207

### CODE REVIEW

**Comment by samtstern** [comment]
I already updated the dependencies in the 7.0 branch here: https://github.com/firebase/FirebaseUI-Android/pull/1853

But the syntactical changes are welcome!

**Comment by rosariopfernandes** [comment]
@samtstern I see. I changed the base branch to `version-7.0.0-dev` in order to get your changes.

File: auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneNumberVerificationHandler.java [hunk 1]
```diff
@@ -27,22 +28,21 @@ public PhoneNumberVerificationHandler(Application application) {
         super(application);
     }
 
-    public void verifyPhoneNumber(@NonNull Activity activity, final String number, boolean force) {
+    public void verifyPhoneNumber(Activity activity, final String number, boolean force) {
```
Comment (rosariopfernandes) [inline auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneNumberVerificationHandler.java:31]: Seems like I removed the annotation when merging the branches

```suggestion
    public void verifyPhoneNumber(@NonNull Activity activity, final String number, boolean force) {
```

**Comment by samtstern** [comment]
@rosariopfernandes LGTM thank you!

**Pull request merged**